src/components/radio/Radio.constants.ts

```diff
@@ -0,0 +1,13 @@
+import { TRadioSize, TRadioSizeToken } from './Radio.types';
```

Comment:
we can add maps here for the sizes used inside the components sub-components as well (same as with the `Checkbox` component)